pyiceberg/manifest.py

```diff
@@ -697,6 +695,15 @@ def content(self) -> ManifestContent: ...
     @abstractmethod
     def version(self) -> TableVersion: ...
 
+    @property
+    def _meta(self) -> Dict[str, str]:
+        return {
+            "schema": self._schema.model_dump_json(),
+            "partition-spec": to_json(self._spec.fields).decode("utf-8"),
```

Comment:
is there a reason why we dont want to use the same logic? Like
```
                "partition-spec": self._spec.model_dump_json(),
```

Reply:
Unfortunately the spec.fields returns a list, which is not a Pydantic object, but a native Python construct. So the method isn't available.